worker/masking/sam2_pose_masker.py

```diff
@@ -381,6 +381,8 @@ def _fine_tune_bounding_box(self, bbox, frame_width, frame_height):
         return [np.int64(x_min), np.int64(y_min), np.int64(x_max), np.int64(y_max)]
 
     def _render_all_masks_on_image(self, image, mask_renderers, frame_idx, masks):
+        if frame_idx not in masks: # in case person is not in 1st frame
```

Comment:
Nitpicking: I think the comment could be a bit vague. Maybe "If the 1st prompt is not on the 1st frame, there won't be any masks until the 1st prompt"